src/routes/gallery/lib/gallery.ts

```diff
@@ -127,24 +123,24 @@ export const uploadImageToWNFS: (
     const { selectedArea } = getStore(galleryStore)
     const fs = getStore(filesystemStore)
 
-    // Reject files over 5MB
+    // Reject files over 20MB
     const imageSizeInMB = image.size / (1024 * 1024)
     if (imageSizeInMB > FILE_SIZE_LIMIT) {
       throw new Error('Image can be no larger than 5MB')
```

Comment:
I'll update this error to say `20MB`. I'll also update the avatar upload to have the same limit 👍🏼